.version

```diff
@@ -1 +1 @@
-3.0.0
+3.1.0
```

Comment:
Im going to be sneaky here...
keep the version at 3.0.0

We will retag.

This is 'dangerous' but works b/c we know EXACTLY whats using this.

Reply:
Reverted back to 3.0.0.